### DISCUSSION TITLE
[4.x] extend email template to other component

### DISCUSSION THREAD

**Discussion by micker** [joomla/joomla-cms#36646] [2. Cold Features]
Hello
We have workflow manager
We have template email
That realy greeeat
But its a little incomplet feature
Why we can't use it together ?
And for now if i understand we can't use email template in 3dth extension
Imagine
Customise all email sended for each steep of workflow
If component use workflow integrator Can manage all emails ...
Cool feature not ?

**Comment by alexandreelise** [comment]
@micker Yup would be pretty cool

**Comment by roland-d** [comment]
@micker That would be a really nice implementation because now workflow uses just a text string. So the first step would be to make the email templates cross extension like workflow and categories. That would be a great feature for 4.2. Would you like to work on that?

**Reply by micker** [reply]
Hello iam not a dev .... Sorry for that...but i Can help for Idea and testing
For me we need to add a context for each component and a filter by component in email list
After an example plugin for text remplacement (like {tittle} etc) to help component conceptor
